.github/workflows/verify_unit_test.yml

```diff
@@ -0,0 +1,24 @@
+name: Verify Unit tests
+
+on:
+  pull_request:
+    paths:
+      - 'support/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: v1.19
+        import-path: github.com/project-codeflare/codeflare-common
```

Comment:
Is this property needed? I guess this should be default path for go.mod/go.sum.

Reply:
It's set as a default in the go.mod and go.sum files, and I'll be working on those.